src/store/tests/vehicle-details/cat-adi-part3/vehicle-details.cat-adi-part3.reducer.ts

```diff
@@ -0,0 +1,29 @@
+import { VehicleDetails } from '@dvsa/mes-test-schema/categories/ADI3';
+import { createFeatureSelector, createReducer, on } from '@ngrx/store';
+import * as vehicleDetailsActions from '../vehicle-details.actions';
+
+const initialState: VehicleDetails = {
+  registrationNumber: '',
+};
+
+export const vehicleDetailsCatADIPart3Reducer = createReducer(
+  initialState,
+  on(vehicleDetailsActions.VehicleRegistrationChanged, (state, { registrationNumber }): VehicleDetails => ({
+    ...state,
+    registrationNumber,
+  })),
+  on(vehicleDetailsActions.DualControlsToggled, (state): VehicleDetails => ({
+    ...state,
+    dualControls: !state.dualControls,
```

Comment:
is toggle correct here? as this is different from other dual control data capture in the app a DualControlsChanged is more approprate? as setting it with a NOT statement (!) will fail with an initial state of null, no?

Reply:
I can update to an explicit ```dualControls: true``` &  ```dualControls: false``` if we prefer this?